db/seeds/test_data.rb

```diff
@@ -408,81 +444,117 @@
   u.full_name = "FIP Mentor"
 end
 teacher_profile = TeacherProfile.find_or_create_by!(user: user)
-ParticipantProfile::Mentor.find_or_create_by!(teacher_profile: teacher_profile) do |mentor_profile|
+participant_profile = ParticipantProfile::Mentor.find_or_create_by!(teacher_profile: teacher_profile) do |mentor_profile|
   mentor_profile.school_cohort = School.find_by(urn: "000105").school_cohorts.find_by(cohort: Cohort.current)
   mentor_profile.schedule = Finance::Schedule::ECF.default
   mentor_profile.participant_identity = Identity::Create.call(user: user, origin: :ecf)
   ParticipantProfileState.find_or_create_by!(participant_profile: mentor_profile)
 end
 
+induction_programme = participant_profile.school_cohort.induction_programmes.first
```

Comment:
Should we add an order close, just for preventive good measure? seems there will be only one at this point so maybe not necessary?

Reply:
i checked these, there only one. i ran the script 20 to makes there weren't any oddities.